src/Commands/ConveyorControl.cpp

```diff
@@ -2,7 +2,7 @@
 #include "../Subsystems/BallConveyor.h"
 
 ConveyorControl::ConveyorControl(double conveyorSpeed, double moverSpeed) :
-  mConveyorSpeed(conveyorSpeed), mMoverSpeed(moverSpeed)
+  mConveyorSpeed(conveyorSpeed)
```

Comment:
Now that the conveyor is one motor it should really be a SetConveyor and not ConveyorControl since that was necessary for its earlier multiple motors. That might not be a reasonable at this point however.